README.md

```diff
@@ -123,6 +123,7 @@ In applications:
 - [Eigen](http://eigen.tuxfamily.org) For linear solvers used in the [LinearSolversApplication](applications/LinearSolversApplication)
 - [Trilinos](https://trilinos.org/) for MPI linear algebra and solvers used in [TrilinosApplication](applications/TrilinosApplication)
 - [METIS](http://glaros.dtc.umn.edu/gkhome/views/metis) for partitioning in [MetisApplication](applications/MetisApplication/README.md)
+- [CoSimIO](https://github.com/KratosMultiphysics/CoSimIO) for performing coupled simulations with external solvers within the [CoSimulationApplication](applications/CoSimulationApplication/README.md). Check [here](https://github.com/KratosMultiphysics/CoSimIO#special-thanks-to) to see the libraries that are used by the CoSimIO.
```

Comment:
I would add the libraries directly here. Like the Egien or Trilinos.

Reply:
done